smartsim/experiment.py

```diff
@@ -878,3 +881,35 @@ def append_to_db_identifier_list(self, db_identifier: str) -> None:
             )
         # Otherwise, add
         self.db_identifiers.add(db_identifier)
+
+    def enable_telemetry(self) -> None:
+        """Experiments will start producing telemetry for all entities run
+        through ``Experiment.start``
+
+        .. warning::
+
+            This method is currently implemented so that ALL ``Experiment``
+            instances will begin producing telemetry data. In the future it
+            planned to have this method work on a "per instance" basis!
```

Comment:
Minor: "it planned" -> "it is planned"

Reply:
fixed!

Reply:
Can you all tell that I have set up a spell checker, but not a grammar checker in my IDE 🙃